src/replit_river/codegen/client.py

```diff
@@ -882,14 +886,16 @@ def __init__(self, client: river.Client[Any]):
         #     the function strings in the branches below, otherwise `dedent`
         #     will pick our indentation level for normalization, which will
         #     break the "def" indentation presuppositions.
+        ottd_name = render_literal_type(output_type_type_adapter_name)
         parse_output_method = f"""\
-                            lambda x: {output_type_type_adapter_name.value}
+                            lambda x: {ottd_name}
```

Comment:
avoid a11s please, it hampers readability

Reply:
makes the line too long here otherwise